src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -148,32 +151,29 @@ public ConnectionBuilder<TSourceType> Bidirectional()
             }
 
             Argument<StringGraphType, string>("before",
-                "Only look at connected edges with cursors smaller than the value of `before`.");
+                "Only look at connected edges with cursors smaller than the value of 'before'.");
             Argument<IntGraphType, int?>("last",
-                "Specifies the number of edges to return counting reversely from `before`, or the last entry if `before` is not specified.");
-
-            _isUnidirectional = false;
-            _isBidirectional = true;
+                "Specifies the number of edges to return counting reversely from 'before', or the last entry if 'before' is not specified.");
```

Comment:
```suggestion
                "Specifies the maximum number of edges to return, starting prior to the cursor specified by 'before', or the last number of edges if 'before' is not specified.");
```

Reply:
I think you fixed an error in that comment :) https://relay.dev/graphql/connections.htm#sec-Edge-order , "counting reversely" was wrong.